src/Pie.js

```diff
@@ -156,7 +156,7 @@ const CleanUpCircles = ({dimensions, backgroundColor, visible}) => {
 }
 
 const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, dividerSize }) => {
-  strokeCapForLargeBands = dividerSize > 0 || strokeCap == 'butt' ? 'butt' : 'butt';
```

Comment:
@nihgwu I see what you mean, however this should definitely be a typo, I will pull and push a PR to fix this then, and revert the  `if((width) > 100 && visible){` to `if((width) < 100 && visible){` but this is not fixing the issue number #44.

were you guys able to reproduce this issue on your side?

Reply:
Yea, that ternary doesn't make much sense as it is right now, likely a typo